hrms/payroll/doctype/employee_benefit_ledger/employee_benefit_ledger.json

```diff
@@ -0,0 +1,178 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2025-04-28 01:15:19.179381",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "posting_date",
+  "employee",
+  "employee_name",
+  "salary_component",
+  "column_break_llqa",
+  "company",
+  "payroll_period",
+  "salary_structure_assignment",
+  "salary_slip",
+  "employee_benefit_details_section",
+  "transaction_type",
+  "amount",
+  "flexible_benefit",
+  "column_break_erll",
+  "yearly_benefit",
+  "remarks"
+ ],
+ "fields": [
+  {
+   "fieldname": "employee",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "in_standard_filter": 1,
+   "label": "Employee",
+   "options": "Employee",
+   "search_index": 1
+  },
+  {
+   "fetch_from": "employee.employee_name",
+   "fieldname": "employee_name",
+   "fieldtype": "Data",
+   "label": "Employee Name"
+  },
+  {
+   "fetch_from": "employee.company",
+   "fieldname": "company",
+   "fieldtype": "Data",
+   "label": "Company"
+  },
+  {
+   "fieldname": "transaction_type",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "in_standard_filter": 1,
+   "label": "Transaction Type",
+   "options": "Accrual\nPayout"
+  },
+  {
+   "default": "Today",
+   "fieldname": "posting_date",
+   "fieldtype": "Date",
+   "label": "Posting Date"
+  },
+  {
+   "fieldname": "column_break_llqa",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "employee_benefit_details_section",
+   "fieldtype": "Section Break",
+   "label": "Employee Benefit Details"
```

Comment:
```suggestion
   "label": "Benefit Details"
```
Can skip reusing the word "Employee" everywhere since the form title already sets that context